feature/google/src/main/java/com/revenuecat/purchases/google/BillingWrapper.kt

```diff
@@ -217,6 +218,7 @@ class BillingWrapper(
         synchronized(this@BillingWrapper) {
             productTypes[storeProduct.productId] = storeProduct.type
             presentedOfferingsByProductIdentifier[storeProduct.productId] = presentedOfferingIdentifier
+            purchaseOptionSelectedByProductIdentifier[storeProduct.productId] = purchaseOption?.id
```

Comment:
I think purchaseOption can be either baseplan_id or offer_id and they don't have to be unique, so I think we need to change this:
![image](https://user-images.githubusercontent.com/273236/203545443-14a0a12a-879a-41a4-b712-035406c47a62.png)

Reply:
i'm not sure i understand. our `PurchaseOption.id` will be the base plan ID if the purchase option is simply a base plan, or it will be the offer id if it's an offer. 

i might be misunderstanding what you mean.

Reply:
ah yes, i see what you mean. i will think about that....